README.md

```diff
@@ -88,6 +89,11 @@ To run the project, execute the following command:
   $ rails server
 ```
 
+### Run tests <a name="run-tests"></a>
+``` 
+  $ rspec
+```
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] It will be great to ensure that the MIT license is not broken. At the moment, it is showing 404 pages. ⬇️ 
<img width="1597" alt="Screenshot 2023-08-18 at 11 07 33" src="https://github.com/lenkon/Blog-App-Ruby-on-Rails/assets/32361963/d49da661-0e79-4084-862b-9424fae487ff">

Reply:
Hi @peterrobert,

Thanks for the feedback. I've made the required changes.

Best regards,
K. Inoulen Shanongba